content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+## Overview
+Datadog Synthetics monitors your applications and API endpoints via simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+This guide shows you how to set up your first Synthetics test with Datadog. Follow the sections below to configure your API, browser test, or private location for internal-facing apps or private URLs.
+
+## Prerequisites
+
+The browser tests can be only recorded from **[Google Chrome][1]**. Datadog will prompt you to download this extension during the creation of the test. You can also download the [Datadog Record Test extension for Google Chrome][2].
+
+## Create a Datadog account
+
+If you haven't already, create a [Datadog account][3].
+
+## Configure your first test
+
+Synthetics offers three different ways to monitor your applications and API endpoints: API test, browser test, and private locations.
+
+### Create an API test
+
+[API tests][4] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][5] to see the full list.
+
+To configure an API test:
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test.
+
+{{< tabs >}}
+
+{{% tab "HTTP Test" %}}
+
+{{< img src="synthetics/api_tests/make-http-request.png" alt="Make HTTP Request" responsive="true" style="width:80%;" >}}
+
+Define the request you want to be executed by Datadog:
+
+1. **Choose request type**: `HTTP`
+2. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+3. **Name**: Set the name of your API test.
+4. **Select your tags**: Select the tags attached to your API test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetics page][1].
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.
```

Comment:
> You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.

Are we redirecting to https://docs.datadoghq.com/synthetics/private_locations/#overview on purpose? Wouldn't it make more sense to redirect to the getting started private location section? Just asking really